testing/simapp/test_helpers.go

```diff
@@ -209,8 +208,8 @@ func SetupWithGenesisAccounts(genAccs []authtypes.GenesisAccount, balances ...ba
 //
 // CONTRACT: BeginBlock must be called before this function.
 func SignAndDeliver(
```

Comment:
sorry to say this is API breaking! We need to revert and readjust the tests

Reply:
Ah dang, ok. I'll get to work

Reply:
This is resolved now!